addons/packages/pinniped/0.22.0/README.md

```diff
@@ -0,0 +1,162 @@
+| `dex.dns.vsphere.ipAddresses` |  `[]`  |
+| `dex.image.name` |  `DEPRECATED`  |
+| `dex.image.pullPolicy` |  `DEPRECATED`  |
+| `dex.image.repository` |  `DEPRECATED`  |
+| `dex.image.tag` |  `DEPRECATED`  |
+| `dex.namespace` |  `tanzu-system-auth`  |
+| `dex.organization` |  `vmware`  |
+| `dex.service.name` |  `dexsvc`  |
+| `dex.service.type` |  `<nil>`  |
+| `http_proxy` |  |
+| `https_proxy` |  |
+| `identity_management_type` |  `<nil>`  |
+| `imageInfo.imagePullPolicy` |  `IfNotPresent`  |
+| `imageInfo.imageRepository` |  `projects-stg.registry.vmware.com/tkg`  |
+| `imageInfo.images.dexImage.imagePath` |  `dex`  |
+| `imageInfo.images.dexImage.tag` |  `v2.27.0_vmware.1`  |
+| `imageInfo.images.pinnipedImage.imagePath` |  `pinniped`  |
+| `imageInfo.images.pinnipedImage.tag` |  `v0.4.1_vmware.1`  |
+| `imageInfo.images.tkgPinnipedPostDeployImage.imagePath` |  `tkg-pinniped-post-deploy`  |
+| `imageInfo.images.tkgPinnipedPostDeployImage.tag` |  `v0.4.1_vmware.1`  |
+| `infrastructure_provider` |  `<nil>`  |
+| `no_proxy` |  |
+| `nodeSelector` |  `<nil>`  |
+| `pinniped.cert_duration` |  `2160h`  |
+| `pinniped.cert_renew_before` |  `360h`  |
+| `pinniped.concierge.audience` |  `<nil>`  |
+| `pinniped.image.name` |  `DEPRECATED`  |
+| `pinniped.image.pull_policy` |  `DEPRECATED`  |
+| `pinniped.image.repository` |  `DEPRECATED`  |
+| `pinniped.image.tag` |  `DEPRECATED`  |
+| `pinniped.post_deploy_job_image.name` |  `DEPRECATED`  |
+| `pinniped.post_deploy_job_image.pull_policy` |  `DEPRECATED`  |
+| `pinniped.post_deploy_job_image.repository` |  `DEPRECATED`  |
+| `pinniped.post_deploy_job_image.tag` |  `DEPRECATED`  |
+| `pinniped.supervisor.service.name` |  `pinniped-supervisor`  |
+| `pinniped.supervisor.service.type` |  `<nil>`  |
+| `pinniped.supervisor_ca_bundle_data` |  `ca_bundle_data_of_pinniped_supervisor_svc`  |
+| `pinniped.supervisor_svc_endpoint` |  `https://0.0.0.0:31234`  |
+| `pinniped.supervisor_svc_external_dns` |  `<nil>`  |
+| `pinniped.supervisor_svc_external_ip` |  `0.0.0.0`  |
+| `pinniped.upstream_oidc_additional_scopes` |  `[]`  |
+| `pinniped.upstream_oidc_claims.groups` |  |
+| `pinniped.upstream_oidc_claims.username` |  |
+| `pinniped.upstream_oidc_client_id` |  |
+| `pinniped.upstream_oidc_client_secret` |  |
+| `pinniped.upstream_oidc_issuer_url` |  `https://0.0.0.0:30167`  |
+| `pinniped.upstream_oidc_provider_name` |  `DEPRECATED`  |
+| `pinniped.upstream_oidc_tls_ca_data` |  `ca_bundle_data_of_dex_svc`  |
+| `tkg_cluster_role` |  `<nil>`  |
+
+## Usage Example
+
+See bundle/examples directory for example configurations of the Pinniped package.
+
+## Building the templates
+
+Build the templates using `oidc` or `ldap` overlay:
+
+```bash
+cd addons/packages/pinniped/0.22.0/bundle
+ytt -f config -f examples/mc-oidc.yaml
+```
+
+## Generate image package
+
+`kbld` will generate the `bundle/.imgpkg/images.yml` file via the following:
+
+```bash
+cd addons/packages/pinniped/0.22.0/bundle
```

Comment:
@joshuatcasey do this and I think our `imgpkg` bits will be closer to correct.

Reply:
Done.